server/src/main/java/org/candlepin/util/ServiceLevelValidator.java

```diff
@@ -72,7 +71,7 @@ public void validate(String ownerId, Collection<String> serviceLevels) {
             String error = i18n.tr("Service level \"{0}\" is not available to units of organization {1}.",
                 StringUtils.join(invalidServiceLevels, ", "), owner.getKey());
 
-            throw new BadRequestException(error);
+            throw new RuleValidationException(error);
```

Comment:
This is a bit confusing place for this exception, since it doesn't come from any rules (`policy` package) execution. BadRequestException seemd a better fit.

Reply:
Changed to SlaValidationException.